widgets/filters/src/FilterActions.ts

```diff
@@ -14,4 +14,14 @@ export default class FilterActions {
     doGetFilterUsage(filterId: string): Promise<FilterUsage[]> {
         return this.toolbox.getInternal().doGet(`/filters/usage/${filterId}`);
     }
+
+    doCreate(filterId: string, filterRules: []) {
```

Comment:
Optional:

The empty array type here is actually an empty tuple, meaning that this function will only accept empty arrays :smile: Not sure if that was what you're going for :slightly_smiling_face: 

Using

```suggestion
    doCreate(filterId: string, filterRules: unknown[]) {
```

would probably be a bit nicer to use :slightly_smiling_face: I understand it's only an intermediate state, so no changes necessary here

IMO it would be nice to use the `FilterRule` type added by @qooban in #1284 

```suggestion
    doCreate(filterId: string, filterRules: FilterRule[]) {
```

---

The same applies to the `doUpdate` method

Reply:
I reused the FilterRule type.
I used direct type import from `common/src/filters/types` - probably temporary solution until we agree where to store REST types and how to access them. For now as it is just a type import I assume it is safe.

Reply:
Ok sure :slightly_smiling_face: